reaktive/src/commonMain/kotlin/com/badoo/reaktive/observable/Interval.kt

```diff
@@ -0,0 +1,19 @@
+package com.badoo.reaktive.observable
+
+import com.badoo.reaktive.scheduler.Scheduler
+import com.badoo.reaktive.utils.atomic.AtomicLong
+
+fun observableInterval(startDelayMillis: Long, periodMillis: Long, scheduler: Scheduler): Observable<Long> =
+    observable { emitter ->
+        val executor = scheduler.newExecutor()
+        emitter.setDisposable(executor)
+
+        val count = AtomicLong()
+        executor.submitRepeating(startDelayMillis, periodMillis) {
+            emitter.onNext(count.value)
```

Comment:
Let's initialize AtomicLong with -1L and then just `emitter.onNext(count.addAndGet(1L))`. It's better to use its return value instead of asking separately.